src/Npgsql/NpgsqlConnectionStringBuilder.cs

```diff
@@ -1808,6 +1827,25 @@ public enum SslMode
     VerifyFull
 }
 
+/// <summary>
+/// Specifies how to manage channel binding.
+/// </summary>
+public enum ChannelBinding
+{
+    /// <summary>
+    /// Channel binding is disabled. If the server requires channel binding, the connection will fail.
```

Comment:
Is this accurate? I thought channel binding is something that the client requires (or disables), not something the server requires.

Reply:
PG returns to us the list of supported methods which might contain only scram-sha256-plus which does require channel binding (I think that list is specified in the source code). Might be some other way to force only scram-sha256-plus (pg_hba.conf?) though I haven't investigated yet.

Reply:
OK. Just mentioning that the libpq option docs [clearly says](https://www.postgresql.org/docs/current/libpq-connect.html) that this is about the "client's use of channel binding", rather than the server requiring anything.